### ISSUE TITLE
[iOS] VoiceOver reads Stats cards incorrectly 

### ISSUE THREAD

**Issue by guarani** [wordpress-mobile/WordPress-iOS#18993]
### Expected behavior

When VoiceOver is used on the Stats Insights screen, the information on the screen should be read similar to how a sighted user would read the data off the screen.

### Actual behavior

VoiceOver on Stats Insights doesn't read data in a way that makes sense for the user.

### Screenshots 

| <img src="https://user-images.githubusercontent.com/1898325/177415827-106b0a73-b0e0-4e91-8535-3817655296f8.png" alt="" width="350">  |
| - |

_Thank you @tiagomar for reporting the issue and making the above image explaining the problem! ❤️_

### Steps to reproduce the behavior

1. Navigate to the Stats Insights screen 
2. Ensure the `All-Time`, `This Year`, and `Today` cards are shown (add them via the gear button if needed)
3. Turn on VoiceOver
4. Tap the `All-Time` card
5. Notice that VoiceOver reads the card as "All-Time Views Visitors 4,182 633..." instead of "All-Time Views 4,182 Visitors 633"
6. Repeat steps 4 to 5 for the other aforementioned cards

https://github.com/Automattic/Temporary-Private-Repo/issues/1

##### Tested on iPhone 11, iOS 15.5, WPiOS 20.2
